scripts/lsh.py

```diff
@@ -127,70 +132,93 @@ def deduplicate_self(file_prefix: str, output_dir: str,
     return bw.total_written, total_read
 
 
-def deduplicate_other(main_batch: str,
-                      batches_to_subtract: list[str],
-                      output_dir: str,
+def read_batch_to_lsh(batch: Path, threshold: float, permutations: int) \
```

Comment:
I'd rather have the arguments on a separate, indented line, and then the last line would be `) -> MinHashLSH:`. I must admit I am not fond of using `\`s in proper code.

Reply:
Me neither, but flake8 is bugging me for having lines too long... I changed it.